### PR TITLE
Fix/RLP-823/signed-unlock

### PR DESCRIPTION
**Pull request by mauricioirace**
Changes:
- Added signature check to prevent unlock transaction to be sent twice

### CODE REVIEW

File: raiden/api/rest.py [hunk 1]
```diff
@@ -1671,6 +1673,32 @@ def settlement_light(self,
...
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Light Client Message with internal_msg_identifier = {} not found"
+                       .format(internal_msg_identifier),
+                status_code=HTTPStatus.BAD_REQUEST,
+                log=log
+            )
+
+        if message.is_signed:
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Failed trying to settle a channel that's already settled",
```
Comment (JONAF2103) [inline raiden/api/rest.py]: All these changes are from another already merged PR, be careful with this.

File: raiden/api/v1/encoding.py [hunk 2]
```diff
@@ -489,6 +489,7 @@ class Meta:
 class SettlementLightSchema(BaseSchema):
     signed_settle_tx = fields.String(required=True)
     channel_identifier = fields.Integer(required=True)
+    internal_msg_identifier = fields.Integer(required=True)
```
Comment (JONAF2103) [inline raiden/api/v1/encoding.py:492]: Same here

File: raiden/api/webui/static/endpointConfig.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const backendUrl='http://localhost:5002'; 
-const nodeAddress = '0xBA7685103a328f5D4509C6C642deFDA4F4fC1A2a'; 
+const nodeAddress = '0x66335087fF02A82d50b9fd81aC268c56c8aE263a';
```
Comment (JONAF2103) [inline raiden/api/webui/static/endpointConfig.js]: Remove this change

File: raiden/lightclient/handlers/light_client_message_handler.py [hunk 4]
```diff
@@ -54,21 +54,30 @@ def store_light_client_protocol_message(cls, identifier: int, message: Message,
         )
 
     @classmethod
-    def update_stored_msg_set_signed_data(
+    def update_offchain_light_client_protocol_message_set_signed_message(
```
Comment (JONAF2103) [inline raiden/lightclient/handlers/light_client_message_handler.py]: Same here, another PR has this already merged on develop

**Review by JONAF2103 — CHANGES_REQUESTED** [review]
I see a lot of changes that are from another PR, please update this with develop content first to get rid of the extra changes that are not related to this

File: raiden/api/v1/encoding.py (hunk 2 above)
Comment (mauricioirace) [reply raiden/api/v1/encoding.py:492]: There are no changes in this lines

File: raiden/api/rest.py [hunk 5]
```diff
@@ -893,7 +893,7 @@ def leave(self, registry_address: typing.PaymentNetworkID, token_address: typing
     @requires_api_key
     def register_secret_light(self, signed_tx: typing.SignedTransaction):
         try:
-            self.raiden_api.register_secret_light(signed_tx)
+            self.raiden_api.register_secret_light(signed_tx)git merg
```
Comment (JONAF2103) [inline raiden/api/rest.py]: Typo?

**Review by JONAF2103 — CHANGES_REQUESTED** [review]
LGTM in general but you need to fix a typo that you have there before i can approve this

**Review by JONAF2103 — APPROVED** [review]
LGTM

**Review by mortelli — APPROVED** [review]

**Pull request merged**